src/core/osmLayer.js

```diff
@@ -441,23 +429,58 @@ geo.osmLayer = function (arg) {
     worldPt2[1] = Math.max(worldPt2[1], -180.0);
     worldPt2[1] = Math.min(worldPt2[1], 180.0);
 
+    /// Compute tile zoom
+    worldDelta = Math.abs(worldPt2[0] - worldPt1[0]);
+    worldDeltaY = Math.abs(worldPt2[1] - worldPt1[1]);
+
+    displayDeltaX = urx - llx;
+    displayDeltaY = lly - ury;
+
+    /// Reuse variables
+    if (displayDeltaY > displayDeltaX) {
+      displayDeltaX = displayDeltaY;
+      worldDelta = worldDeltaY;
+    }
+
+    if (Math.log2) {
```

Comment:
We already have a polyfill for this.  https://github.com/OpenGeoscience/geojs/blob/master/src/core/init.js#L219